auth-manifest/README.md

```diff
@@ -3,37 +3,50 @@
+| Image Identifier        | 4            | Unique value selected by the vendor to distinguish between component images.<br />IMEs should be sorted by an ascending order of Image Identifier within an IMC.                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| Flags                   | 4            | A set of 32-bit flags specific to the image.<br />**Bit0**: If set, the image hash will not be verified otherwise the metadata image hash will be compared against the calculated hash of the image.<br />**Bit1**: If set, it indicates that the image is a MCU Runtime image, otherwise it indicates a SOC Image.                                                                                                                                                                                                                                                  |
+| Image Load Address High | 4            | The higher 4-bytes of the 64-bit AXI address where the image will be loaded for verification and execution or staging.                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| Image Load Address Low  | 4            | The lower 4-bytes of the 64-bit AXI address where the image will be loaded for verification and execution or staging                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
nit: "4-byte" (with hyphen) is for when it is an adjective, but "4 bytes" is for when it is a noun.

```suggestion
| Image Load Address High | 4            | The higher 4 bytes of the 64-bit AXI address where the image will be loaded for verification and execution or staging.                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
| Image Load Address Low  | 4            | The lower 4 bytes of the 64-bit AXI address where the image will be loaded for verification and execution or staging                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```